proto/regen/ecocredit/marketplace/v1beta1/state.proto

```diff
@@ -113,16 +113,16 @@ message BuyOrder {
     bool maker = 9;
 }
 
-// AllowedDenom represents the information for an allowed ask/bid denom.
+// AllowedDenom represents the information for an allowed ask/bid base denom.
 message AllowedDenom {
     option (cosmos.orm.v1alpha1.table) = {
         id: 3
-        primary_key: {fields: "bank_denom"}
+        primary_key: {fields: "base_denom"}
         index: {id: 1 fields: "display_denom", unique: true}
     };
 
-    // denom is the bank denom to allow (ex. ibc/GLKHDSG423SGS)
-    string bank_denom = 1;
+    // base is the bank denom to allow (ex. ibc/GLKHDSG423SGS)
+    string base_denom = 1;
```

Comment:
could be `base_currency` or `base_token`

Reply:
base_currecny or currency_denom. But honestly I don't see this as a big deal

Reply:
base_denom is less clear to me though